apps/vite/src/components/AddEventModal.tsx

```
@@ -10,19 +10,20 @@ import colors from "../constants/colors"
 import { Switch } from "@headlessui/react"
 import NepaliDatePicker from "./NepaliDatePicker"
 import { useQueryClient, useMutation, useQuery } from "@tanstack/react-query"
-import { CalendarEvent } from "../types/events.types"
-import { toast } from "react-hot-toast"
-import { apiBaseUrl, createEvent, getCalendarList } from "../helper/api"
+import { CalendarEvent } from "@miti/types"
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)